sanic/app.py

```diff
@@ -220,6 +223,12 @@ def __init__(
         self.go_fast = self.run
 
         if register is not None:
+            warn(
+                "The register argument is deprecated and will stop working "
+                "in vXX.X. After vXX.X all apps will be added to the Sanic "
+                "app registry.",
+                DeprecationWarning,
+            )
```

Comment:
Maybe not the right PR for this. But, I suppose we can sneak it in here instead. v22.6

Reply:
I can submit a separate PR if you wish?

Reply:
belongs here https://github.com/sanic-org/sanic/pull/2306